libs/anglify/src/lib/modules/radio-button/radio-button.interface.ts

```diff
@@ -0,0 +1,13 @@
+import type { BooleanLike } from '../../utils/interfaces';
+
+export interface RadioButtonSettings {
+  disabled?: BooleanLike;
+  checked?: BooleanLike;
+  ripple?: BooleanLike;
+  labelPosition?: RadioLabelPosition;
+  rippleOrigin?: RadioRippleOrigin;
+}
+
+export type RadioLabelPosition = 'before' | 'after';
+
+export type RadioRippleOrigin = 'center' | undefined;
```

Comment:
Can you please rename this type to `OverlayRippleOrigin` and move it into the Overlay module?
In addition, it would be great if you could add the type to the input property `rippleOrigin` within the overlay directive. In the checkbox module this also needs to be refactored, because the checkbox has an identical interface.

Thanks 🙂

Reply:
Done.